ivy/functional/frontends/jax/numpy/creation.py

```diff
@@ -1,7 +1,7 @@
 import ivy
 from ivy.func_wrapper import with_unsupported_dtypes
 from ivy.functional.frontends.jax.array import Array
-import ivy.functional.frontends.jax.numpy as jnp_frontend
+import ivy.functional.frontends.jax.numpy as jnp
```

Comment:
if we have to change the name of the import at either of the 2 places, I'd much rather do that in the test file instead.  Sorry for the change of mind, could you please change it so that we do the `jnp` import in the test file and `jnp_frontend` import in the frontend file? Thanks @Vismay-dev 😄

Reply:
it's the front end file import that's causing the error rather than the test file. Not the import itself but the use of this namespace "jnp_frontend" in the function "setdiff1d". 

This line https://github.com/unifyai/ivy/blob/d382882aafb0e19cccc8145cfcaa6cc4f53a77c4/ivy/functional/ivy/data_type.py#L192 checks for the keyword "frontend" in the base names of all the functions invoked in the frontend function "setdiff1d".

Changing the import in the test file, as you originally pointed out, is not related to the error.

Reply:
let's probably change the import in the frontend file to something like `from ivy.functional.frontends.jax import where, unique` similar to the other imports in the `jax` frontend if that fixes the issue 😄

Reply:
@vedpatwardhan that's causing a circular import error. the reason there's no circular import error for the other imports in the `jax` frontend is because those imported functions (for eg. `promote_types_of_jax_inputs`) have been defined in `__init__.py` for the `frontends/jax/numpy` frontend directory.

Reply:
Just pushed a change to `data_type.py`, does test collection now work with `jnp_frontend` in the frontend file?

Reply:
Yes the test collection works with `jnp_frontend` now 😄

Reply:
awesome! We can get the PR merged after the changes to the `demos` submodule are reverted. Feel free to request a review once you're done with that, thanks!

Reply:
@vedpatwardhan all done!